apps/example-todo-app/next.config.js

```diff
@@ -1,15 +1,20 @@
 // This file changes the routing to allow top-level prefixes
 
-module.exports = {
+/** @type {import('next').NextConfig} */
+const nextConfig = {
+  reactStrictMode: true,
   async rewrites() {
     return {
       beforeFiles: [
         // Nextjs by default requires a /api prefix, let's remove that
-        {
-          source: "/:path*",
-          destination: "/api/:path*",
-        },
+        // REVIEW: I was not able to make the redirect work
```

Comment:
I could not make the redirect work, maybe there is some issue with having /app and /pages simultaneously.

Reply:
yea it's ok to not have /pages and /app simultaneously 👍 we should support app directory but that's non-critical for now